examples/graphql-service-basic-auth-file-user-store/graphql_service_basic_auth_file_user_store.md

```diff
@@ -1,4 +1,4 @@
-# GraphQL service - Basic Auth file user store
+# GraphQL service - Basic Authentication file user store
```

Comment:
I think this would be better `Basic authentication - File user store`, `Basic authentication - LDAP user store

Reply:
Fixed